#
# Rollback this offending patch, see
# http://bugs.sourcemage.org/show_bug.cgi?id=15921
#
commit d0cbf388919364fe6b5b9127f36426eb744090a0
Author: Jeremy Huddleston <jeremyhu@apple.com>
Date:   Sun Sep 26 21:21:16 2010 -0700

    Add an "X11_" string to header guards to avoid possible collision

    This addresses a build failure which can result from <X11/Xlocale.h> and
    <xlocale.h> being included in the same code since they both used the same
    _XLOCALE_H_ protection.

    Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>

diff -Naur libX11-1.4.0.orig/include/X11/Xlib.h libX11-1.4.0/include/X11/Xlib.h
--- libX11-1.4.0.orig/include/X11/Xlib.h	2010-12-16 14:19:08.398837832 -0800
+++ libX11-1.4.0/include/X11/Xlib.h	2010-12-16 14:20:17.853837890 -0800
@@ -30,8 +30,8 @@
  *	interface library (Xlib) to the X Window System Protocol (V11).
  *	Structures and symbols starting with "_" are private to the library.
  */
-#ifndef _X11_XLIB_H_
-#define _X11_XLIB_H_
+#ifndef _XLIB_H_
+#define _XLIB_H_
 
 #define XlibSpecificationRelease 6
 
@@ -4020,4 +4020,4 @@
 
 _XFUNCPROTOEND
 
-#endif /* _X11_XLIB_H_ */
+#endif /* _XLIB_H_ */
